src/main.rs

```diff
@@ -219,7 +220,7 @@ async fn application_trampoline(config: &Config) -> Result<()> {
         };
 
         client_builder.set_username(Some(username.into()));
-        client_builder.set_password(Some(password.into()));
+        client_builder.set_password(Some(password.as_bytes().to_vec()));
```

Comment:
I'm not sure this is necessary, I'm new to Rust :)

Reply:
There are several ways to have done it. This is perfectly fine.